src/py/reactpy/reactpy/backend/default.py

```diff
@@ -59,7 +60,11 @@ def _default_implementation() -> BackendImplementation[Any]:
         implementation = next(all_implementations())
     except StopIteration:  # nocov
         logger.debug("Backend implementation import failed", exc_info=exc_info())
-        msg = "No built-in server implementation installed."
+        supported_backends = ', '.join(SUPPORTED_PACKAGES)
+        msg = "It seems you haven't installed an backend. To resolve this issue, you can install a backend by running \n"\
+            "\n" \
+           'pip install "reactpy[starlette]"\n\n'\
+            f"Other supported backends include: {supported_backends}."     
```

Comment:
My recommendation here is to do something like:
```suggestion
        msg = (
            "It seems like you haven't installed a backend. You can fix this issue by running:"
            "\n\n"
            'pip install "reactpy[starlette]"'
            "\n\n"
            f"Other supported backends include: {supported_backends}."
        )
```

Generally line continuations with `\` aren't recommended.

Reply:
Okay! will keep that in mind. Thanks alot